hathor/merged_mining/coordinator.py

```diff
@@ -714,6 +732,40 @@ async def submit_to_bitcoin(self, job: SingleMinerJob, work: SingleMinerWork) ->
             # - unexpected-witness: transaction has inputs with witness but isn't marked as containing witnesses
             self.log.error('block rejected from Bitcoin', reason=res)
 
+    async def submit_to_digibyte(self, job: SingleMinerJob, work: SingleMinerWork) -> None:
+        """ Submit work to DigiByte RPC.
+        """
+        digibyte_rpc = self.coordinator.digibyte_rpc
+        assert digibyte_rpc is not None
+        # bitcoin_block = job.build_bitcoin_block(work)  # XXX: too expensive for now
+        bitcoin_block_header = job.build_parent_block_header(work)
+        block_hash = Hash(bitcoin_block_header.hash)
+        block_target = Target(int.from_bytes(bitcoin_block_header.bits, 'big'))
+        if block_hash.to_u256() > block_target.to_u256():
+            self.log.debug('high hash for DigiByte, keep mining')
+            return
+        if self.coordinator.should_skip_parent_submit(job.parent_height, ParentChain.DIGIBYTE):
+            self.log.debug('late winning share, skipping DigiByte submit')
+            return
+        bitcoin_block = job.build_bitcoin_block(work)  # XXX: far fewer cases, so it's OK now
+        data = bytes(bitcoin_block)
+        try:
+            res = await digibyte_rpc.submit_block(data)
+        except Exception:
+            self.log.warn('submit to Bitcoin failed', exc_info=True)
+            return
+        self.coordinator.update_parent_submitted(job.parent_height, ParentChain.DIGIBYTE)
+        self.log.debug('digibyte_rpc.submit_block', res=res)
+        if res is None:
+            self.log.info('new Bitcoin block found!!!', hash=bitcoin_block.header.hash.hex())
+            await self.coordinator.update_bitcoin_block()
+        else:
+            # Known reasons:
```

Comment:
Are these reasons valid for Digibyte? It seems it was copied from Bitcoin.

Reply:
I updated these with feedback from testing.